ImageTarget[] targets
string name
float32 bearing
